src/main/java/Task.java

```diff
@@ -0,0 +1,29 @@
+class Task {
+    private boolean status = false;
+    private String description;
+    public Task(String description) {
+        this.description = description;
+    }
+    public String getDescription(){
+        return description;
+    }
+    public boolean getStatus() {
+            return status;
+    }
+    public void markAsComplete() {
+        this.status = true;
+        System.out.println("    ____________________________________________________________\n" +
```

Comment:
Can try to separate some of the code as their own methods like the printing of separation lines